AjobK/Athena
# 312 pin comment

closes #312 

<br />

 **What is it supposed to do** 
The owner of posts can pin multiple comments. Comments can also be liked (once). Comments are then ordered, first by whether they are pinned, and then on their amount of likes.
![image](https://user-images.githubusercontent.com/57443004/135755511-c24e38c9-0d90-4130-90ce-1da65782ba23.png)


<br />

 **How can we test your branch**  
1. Create comments on one account
2. Like one of them and see them ordered on refresh
3. Create a post and comment on this post
4. Pin some posts and repeat steps 1 and 2

To test unhappy flow, try pinning posts as a non-post-owner, or try liking comments twice

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
